docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -999,18 +999,30 @@ Quarkus provides a set of built-in extension methods.
 ** `{#if map.isEmpty}`
 
 * `get(key)`: Returns the value to which the specified key is mapped
-** `{map.get('foo')}` or `{map['foo']}`
+** `{map.get('foo')}`
 
-===== Collection extension methods:
+TIP: A map value can be also accessed directly: `{map.myKey}`. Use the bracket notation for keys that are not legal identifiers: `{map['my key']}`.
+
+===== Collections
 
 * `get(index)`: Returns the element at the specified position in a list
-** `{list.get(0)}` or `{list[0]}`
+** `{list.get(0)}`
+
+TIP: A list element can be accessed directly: `{list.10}` or `{list[10]}`.
 
-===== Number extension methods:
+===== Numbers
 
 * `mod`: Modulo operation
 ** `{#if counter.mod(5) == 0}`
 
+===== Config 
+
+* `config:<name>` or `config:[<name>]`: Returns the config value for the given property name
+** `{config:foo}` or `{config:['property.with.dot.in.name']}`
```

Comment:
I'll note that I still prefer to use pre-defined toplevels rather than namespaces, like `config['name']` or `config.name`, because this is less awkward in this situation, but it's not super important.

Reply:
@FroMage I know that you don't like the concept of namespaces... I do ;-). In any case, we might consider adding API to specify pre-defined top-levels I think...

Reply:
That would be very nice. Note that I didn't reject this PR, I'm merely hoping that you keep track of how many times I've pointed out namespace issues and one day you change your mind :)